cogs/music.py

```diff
@@ -19,154 +27,451 @@ def __init__(self, bot: ManChanBot):
+            return
+        embed.title = playlist_name.playlist_name
+        playlist_str = ""
+        index = 1
+        for entry in playlist:
+            song = SongDB.get_by_id(entry.id)
+            if song is not None:
+                playlist_str += (
+                    str(index) + ". " + song.title + "(" + str(song.id) + ")" + "\n"
+                )
+            index += 1
+        embed.add_field("", playlist_str)
+        await ctx.send(embed=embed, delete_after=10)
+
+    @command(aliases=["rls"])
+    async def remove_playlist_song(self, ctx: Context, *args: str):
+        await ctx.message.delete(delay=5)
+        playlist_id = int(args[0])
+        song_id = int(args[1])
+        playlist = PlaylistSongDB.get_songs_by_playlist(playlist_id)
+        if playlist is None:
+            await ctx.send(
+                f"Playlist: {' '.join(args).strip()} does not exist", delete_after=10
+            )
+            return
+        song = SongDB.get_by_id(song_id)
+        if song is None:
+            await ctx.send(f"Song does not exist", delete_after=10)
+            return
+        PlaylistSongDB.delete_playlist_song(playlist_id, song_id)
+        await ctx.send(
+            f"Song {song.title} removed from Playlist: {playlist_id} deleted",
+            delete_after=10,
+        )
+
+    @command(aliases=["ldpl"])
+    async def load_playlist(self, ctx: Context, *args: str):
+        await ctx.message.delete(delay=5)
+        if ctx.guild is None:
+            await ctx.send("This command can only be used in a server.", delete_after=5)
+            return
+        id = int(" ".join(args).strip())
+        playlist = PlaylistSongDB.get_songs_by_playlist(id)
+        if playlist is None:
+            await ctx.send(
+                f"Playlist: {' '.join(args).strip()} does not exist", delete_after=10
+            )
+            return
+        player = self.master_player.get_player(ctx.guild.id)
+        if player is None:
+            return
+        for entry in playlist:
+            songdb = SongDB.get_by_id(entry.id)
+            if songdb is not None:
+                player.add_song(songdb.webpage_url)
+        if player.player_ui is None:
+            return
+        await player.player_ui.edit(
+            embed=self.edit_embed_queue(player, player.player_ui.embeds[0])
+        )
+
+    def default_embed(self) -> disnake.Embed:
+        embed = disnake.Embed()
+        embed.title = "MANCHAN RADIO"
+        embed.description = "Please run !jvc to start radio"
+        embed.add_field("Now Playing:", "Empty", inline=False)
+        embed.add_field("Queue:", "Empty", inline=False)
+        return embed
+
+    def edit_embed_queue(self, player: Player, embed: disnake.Embed) -> disnake.Embed:
+        embed.set_field_at(1, "Queue:", player.queue_to_string(), inline=False)
+        return embed
 
     @classmethod
     def is_enabled(cls, configs: Dict[str, Any] = {}) -> bool:
         return configs["ENABLE_MUSIC"]
```

Comment:
There is now an env that checks if the database is enabled. Music bot will require that now as well